ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -158,6 +158,24 @@ Item {
               color: Theme.palette.directColor1
           }
 
+          MouseArea {
+              property bool accountPopupOpened: false
+
+              anchors.fill: parent
+              anchors.margins: -10
+              cursorShape: Qt.PointingHandCursor
+              hoverEnabled: true
+              onEntered: {
+                  accountPopupOpened = accountsPopup.opened
+              }
+              onPressed: {
```

Comment:
Should be `onClicked`, unless you want to handle the released signal as well

Reply:
Actually yes. It works with onClicked too. I was testing onPressed and onClicked to see difference, but now they work same.

Reply:
Definitely go for `onClicked`